### ISSUE TITLE
Comments.

### ISSUE THREAD

**Issue by madrury** [rakreek/seminar-cnn-for-text#1]
Not too much I would change.

I think you can spare a min to justify the choice of max pooling.  There are three ideas you could hit:

  - Training these things is expensive, so if we can lower the dimensionality of the data as they pass through the network without hurting performance, this would be good..
  - The convolutions we are fitting only see five consecutive words at the first layer.  If we want to build higher level features we would like to expand our range to see more and more of the text fragment.
  - If we, for example, pooled by averaging, then a strong positive presence of the feature could be offset by a strong negative presence, so we would lose some fidelity.  The max pool allow us to say "this thing is happening in this part of the text".

I'm not sure what you're trying to communicate with the picture of the feedforward network.  

I think you are missing a bit of oppurtunity on your hyperparameters slide.  It would be nice to add a couple of words (you may be planning on saying this) about what each of the hyperparameters is trying to accomplish.  For example:

Convolutional Layer
  - Number of filters: More allows the network to capture more features of the text.
Filter width(s): Each filter can depend on more words (or lower level features).
Activation function: ???

I notice it doesnt say convolutional layer*s*... is there a reason for that?  Do people use more than one?

Dropout is a neat idea its worth introducing people to.

That's all.  This looks fun.  I'm going to invite you to give it at galvanize.
